paddle/platform/dynload/dynamic_loader.h

```diff
@@ -58,6 +58,14 @@ void GetWarpCTCDsoHandle(void** dso_handle);
  */
 void GetLapackDsoHandle(void** dso_handle);
 
+/**
+ * @brief    load the DSO of NVIDIA nccl
+ *
+ * @param    **dso_handle   dso handler
+ *
+ */
+void GetNcclDsoHandle(void** dso_handle);
```

Comment:
I am not sure, but should we rename

GetNcclDsoHandle => dso::LoadNCCL
GetLapackDsoHandle => dso::LoadLAPACK